### ISSUE TITLE
Make workers ignore invalid JSON on stdout

### ISSUE THREAD

**Issue by ctrueden** [apposed/appose#2]
Sometimes, workers might want to use libraries that rudely print random things to stdout. When this happens, we shouldn't barf, but rather just carry on with a stiff upper lip. Furthermore, to be safe, every Appose-produced line of valid JSON should include a telltale key/value pair in its dictionary as an obvious sign that the output is in fact from Appose, and not from some other random library.

**Comment by carlosuc3m** [comment]
Hello @ctrueden I have been working quite a lot with Appose recently.
REgarding this issue, doing something as simple as the following works:

```
import groovy.json.JsonSlurper;
private static boolean isJson(String str) {
	try {
		new JsonSlurper().parseText(str);
		return true;
	} catch (Exception ex) {
		return false;
	}
}
```

and then in the `while` loop of `stdoutLoop()` after line https://github.com/apposed/appose-java/blob/b527d884f92d1adfbe482c7a33b91bbf7223de2a/src/main/java/org/apposed/appose/Service.java#L147

```
while (true) {
	String line = stdout.readLine();
	debugService(line == null ? "<worker stdout closed>" : line);

	if (line == null) return; // pipe closed
	if (!isJson(line)) continue;
	...
}
```

With respect to the Key Value pair to indentify Appose encoded messages, could it be something like a String for the key ("appose" for example) and the timestamp of when the class is loaded for the value? It could be transmitted to the Python worker at the initial `RequestType.EXECUTE` request.
If something like that works or you have a clear idea in mind I can do a pull request.

Regards,
Carlos

**Comment by ctrueden** [comment]
Addressed by apposed/appose-java@1b2d79f49869247a7d14d6c888b10f0985134630 and apposed/appose-python@147b2d22e6b88aa2bdd8faae76f26b07142e0249. Regarding a telltale Appose key: already included is `task` with a UUID that must match, which seems sufficient to me.
